app/controllers/admin/dashboard_controller.rb

```diff
@@ -2,7 +2,7 @@ class Admin::DashboardController < ApplicationController
   before_action :authorize!
 
   def index
-    @dashboard_presenter = DashboardPresenter.new
+    
   end
 
   private
```

Comment:
It looks like this dashboard is no longer needed? Authorization should move over to the Admin::CompaniesController now.